prow/config/config.go

```diff
@@ -34,6 +34,13 @@ type Config struct {
 
 	// Periodics are not associated with any repo.
 	Periodics []Periodic `json:"periodics,omitempty"`
+
+	Triggers []Trigger `json:"trigger,omitempty"`
```

Comment:
It is confusing that the member here is `Triggers` but the json key is `trigger`.  This makes the yaml confusing to read.